### DISCUSSION TITLE
cursor improvement

### DISCUSSION THREAD

**Discussion by jabbalaci** [wezterm/wezterm#4545] [Q&A] [answered]
### What Operating System(s) are you running on?

Linux X11

### Which Wayland compositor or X11 Window manager(s) are you using?

Manjaro XFCE

### WezTerm version

wezterm 20230712-072601-f4abf8fd

### Ask your question!

I have the "Terminal Basic" color scheme and it's all good except the cursor. When I type a text, the text appears in black. The block cursor is dark grey. When I move the cursor over a character, the character remains black and the cursor is dark grey -> the character is hardly visible.

In Konsole, in this case, the character under the cursor becomes white, so it's clearly visible.

Question 1: can we mimic the behavior of Konsole?

Question 2: if not, an alternative would be to change the color of the block cursor. For instance, I like the cursor in the color scheme "Mostly Bright (terminal.sexy)". It's green and a letter behind it is visible. How to copy the cursor from a color scheme into another color scheme?

**Comment by arpangreat** [comment, marked as answer]
Locate your theme file and comment out `cursor_fg` and `cursor_bg`. It will mimic the color of the character cell , cursor is on, so basically in blue highlighted text the cursor will change into blue and as follows.

**Comment by jabbalaci** [comment]
Thanks! All I had to do was add these lines to my config file:

```lua
config.colors = {
    cursor_fg = "white",
}
```
